packages/@react-aria/aria-modal-polyfill/test/index.test.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {act, fireEvent, render, waitFor} from '@testing-library/react';
+import {ActionButton, Button} from '@react-spectrum/button';
+import {Content} from '@react-spectrum/view';
+import {Dialog, DialogTrigger} from '@react-spectrum/dialog';
+import MatchMediaMock from 'jest-matchmedia-mock';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import {theme} from '@react-spectrum/theme-default';
+import {triggerPress} from '@react-spectrum/test-utils';
+import {watchModals} from '../';
+
+describe('watchModals', () => {
+  let matchMedia;
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  beforeEach(() => {
+    matchMedia = new MatchMediaMock();
+    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => cb());
+  });
+
+  afterEach(() => {
+    jest.runAllTimers();
+    matchMedia.clear();
+    window.requestAnimationFrame.mockRestore();
+  });
+
+  it('should hide everything except the modal', async () => {
+    watchModals();
+    let {getByLabelText, getByRole} = render(
+      <Provider theme={theme}>
```

Comment:
Would these tests pass without the `watchModals` call? Do we need to render something outside the provider? I believe the provider itself will get `aria-hidden` applied already, without `watchModals`.